src/query-invoice-data.js

```diff
@@ -0,0 +1,58 @@
+const createBaseRequest = require('./create-base-request.js');
+const sendRequest = require('./send-request.js');
+
+/**
+ * Query previously sent invoices with invoice number or query params.
+ * @async
+ * @param {Object} params Function params.
+ * @param {Object} params.invoiceQuery Query single invoice with invoice number.
+ * @param {Object} params.queryParams Query multiple invoices with params.
+ * @param {Object} params.technicalUser Technical user’s data.
+ * @param {Object} params.softwareData Invoice software data.
+ * @param {Object} params.axios Axios instance.
+ * @returns {Promise<Array>} queryResults
+ */
+module.exports = async function queryInvoiceData({
```

Comment:
Both invoiceQuery and queryParams object key order must be normalized before xml conversion.